lib/assessment.js

```diff
@@ -4,23 +4,39 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
+var min = function () {
+  var minimum = arguments[0]
+  for (i=1; i< arguments.length; i++){
+      if (arguments[i] < minimum) {
+        mimumun = arguments][i];
+    }
+  }
+  return minimum;
+}
 
-};
+// I know this is wrong, I need to put args in min(args) but cant remember how to set it to the next variable because when I type in arg = arguments, arugments is already defined and my code will only accept three cases right now because of how its set up as a simple if/then statement.
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-var sum = function sum() {
-
+var sum = function sum(arg1,arg2) {
+    var total = 0;
+    for (var i = 0; i < arguments.length; i++){
+      total += arguments[i];
+    }
+    return total;
```

Comment:
Right on. This function takes named parameters, but actually accesses its parameters through the `arguments` pseudo-array. So it works for any number of addends. 1